### ISSUE TITLE
[Adaptive Cards - Text Block] - In scan mode, screen reader is not conveying the role of the heading using up/down arrow key.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8365]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/explorer/TextBlock.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
1. Turn on the Narrator and turn on the scan mode.

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to all headings in the screen using down arrow key and observe the issue.

**Actual Result:**
In scan mode, while navigating to all headings using down arrow key screen reader is not conveying the role as heading for the labels.

**Expected Result:**
In scan mode, screen reader should convey the role information for the labels, and it should announce the role as 'Heading'.

**Note:**
1. Same issue repro through the NVDA Screen Reader.
2. Same issue is repro throughout the application.

**Observation:**
In scan mode, while navigating using 'H' key, the screen reader is conveying the role as heading.

**User Impact:**
It will impact the screen reader user if the screen reader does not convey the role as heading for the text block label as user won't be able to know whether it is plain text or heading.
 
"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![Usable- Screen reader is not conveying the role as heading in scan mode](https://user-images.githubusercontent.com/95913748/224702833-08275ed6-21a0-44b5-95d6-3066d5726e3c.png)

https://user-images.githubusercontent.com/95913748/224702904-3330e71a-28ee-4790-b4ed-7bbb21946071.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/user-attachments/assets/9ac3704e-864c-4db2-b5f6-41800ad1cdd5)
